### PR TITLE
Update Evidence to Statement Payload

### PR DESCRIPTION
**Pull request by SteveLasker**
Updates Evidence to the payload of a signed statement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ SCITT is intended to store Signed Statements for the life of the of the SCITT in
 
 SCITT is analogous to a digital or electronic notary service (eNotary), where minimal information is written to the append-only log and an augmented counter-signature over the Statement and corresponding metadata (Receipt) is produced. When users notarize legal documents, the notary append-only log records the verified identity of the parties, referencing the legal document they are notarizing. The notary append-only log doesn't store the legal document, but does have a reference to it. 
 
-In SCITT, the append-only log will contain pointers to the artifact, which Statements are made, with optional pointers to any supporting evidence.
+In SCITT, the append-only log will contain pointers to the artifact, which Statements are made, with a payload representing the contents of the statement.
```
Comment (aj-stein-nist) [inline README.md:34]: Are we planning on capitalizing terms from the architecture document as if this is an IETF I-D? I am not sure that helps or hurts people reading this, but we have inconsistent usage here, other changed sections, and around the page.

I am not saying I have a strong opinion one way or the other, I just want to know before reviewing further.

Comment (SteveLasker) [reply README.md:34]: consistency is goodness. I'm torn between doing all Capitizlation in this doc or not. So, will approve any change of consistency

Comment (aj-stein-nist) [reply README.md:34]: OK I'll give it a think and adjust accordingly.

Comment (SteveLasker) [reply README.md:34]: Could we do consistency in a separate PR?

Comment (aj-stein-nist) [reply README.md:34]: Sure, that is fine by me.

**Review by aj-stein-nist — APPROVED** [review]

**Pull request merged**